paparazzi/api/paparazzi.api

```diff
@@ -0,0 +1,259 @@
+}
+
+public abstract interface class app/cash/paparazzi/RenderExtension {
+	public abstract fun renderView (Landroid/view/View;)Landroid/view/View;
+}
+
+public final class app/cash/paparazzi/Snapshot {
+	public static final field $stable I
+	public fun <init> (Ljava/lang/String;Lapp/cash/paparazzi/TestName;Ljava/util/Date;Ljava/util/List;Ljava/lang/String;)V
+	public synthetic fun <init> (Ljava/lang/String;Lapp/cash/paparazzi/TestName;Ljava/util/Date;Ljava/util/List;Ljava/lang/String;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public final fun component1 ()Ljava/lang/String;
+	public final fun component2 ()Lapp/cash/paparazzi/TestName;
+	public final fun component3 ()Ljava/util/Date;
+	public final fun component4 ()Ljava/util/List;
+	public final fun component5 ()Ljava/lang/String;
+	public final fun copy (Ljava/lang/String;Lapp/cash/paparazzi/TestName;Ljava/util/Date;Ljava/util/List;Ljava/lang/String;)Lapp/cash/paparazzi/Snapshot;
+	public static synthetic fun copy$default (Lapp/cash/paparazzi/Snapshot;Ljava/lang/String;Lapp/cash/paparazzi/TestName;Ljava/util/Date;Ljava/util/List;Ljava/lang/String;ILjava/lang/Object;)Lapp/cash/paparazzi/Snapshot;
+	public fun equals (Ljava/lang/Object;)Z
+	public final fun getFile ()Ljava/lang/String;
+	public final fun getName ()Ljava/lang/String;
+	public final fun getTags ()Ljava/util/List;
+	public final fun getTestName ()Lapp/cash/paparazzi/TestName;
+	public final fun getTimestamp ()Ljava/util/Date;
+	public fun hashCode ()I
+	public fun toString ()Ljava/lang/String;
+}
+
+public abstract interface class app/cash/paparazzi/SnapshotHandler : java/io/Closeable {
+	public abstract fun newFrameHandler (Lapp/cash/paparazzi/Snapshot;II)Lapp/cash/paparazzi/SnapshotHandler$FrameHandler;
+}
+
+public abstract interface class app/cash/paparazzi/SnapshotHandler$FrameHandler : java/io/Closeable {
+	public abstract fun handle (Ljava/awt/image/BufferedImage;)V
+}
+
+public final class app/cash/paparazzi/SnapshotJsonAdapter : com/squareup/moshi/JsonAdapter {
+	public static final field $stable I
+	public fun <init> (Lcom/squareup/moshi/Moshi;)V
+	public fun fromJson (Lcom/squareup/moshi/JsonReader;)Lapp/cash/paparazzi/Snapshot;
+	public synthetic fun fromJson (Lcom/squareup/moshi/JsonReader;)Ljava/lang/Object;
+	public fun toJson (Lcom/squareup/moshi/JsonWriter;Lapp/cash/paparazzi/Snapshot;)V
+	public synthetic fun toJson (Lcom/squareup/moshi/JsonWriter;Ljava/lang/Object;)V
+	public fun toString ()Ljava/lang/String;
+}
+
+public final class app/cash/paparazzi/SnapshotVerifier : app/cash/paparazzi/SnapshotHandler {
+	public static final field $stable I
+	public fun <init> (D)V
+	public fun <init> (DLjava/io/File;)V
+	public synthetic fun <init> (DLjava/io/File;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public fun close ()V
+	public fun newFrameHandler (Lapp/cash/paparazzi/Snapshot;II)Lapp/cash/paparazzi/SnapshotHandler$FrameHandler;
+}
+
+public final class app/cash/paparazzi/TestName {
```

Comment:
I think this can be internal

Reply:
not this one unfortunately, as it's a member of Snapshot which is a member of SnapshotHandler which is a public type.

Reply:
Aha, missed that one